cmake/Intel.cmake

```diff
@@ -1,44 +1,13 @@
+set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -fpp -fno-alias -auto -safe-cray-ptr -ftz -assume byterecl -nowarn -sox -align array64byte -qno-opt-dynamic-align")
```

Comment:
can we add `-g -traceback` here and remove them from everywhere else.
These have no performance penalties and are very useful when the execution abruptly fails.